frontend/src/i18n/en/translation.json

```diff
@@ -324,7 +324,7 @@
     "value": "Value",
     "synonyms": "Synonyms",
     "lookups": "Lookups",
-    "lookup_strategies": "Lookup Strategy",
+    "lookup_strategies": "Type",
```

Comment:
Amazing ! 🥳 Thanks for your contribution.

lookup_strategies is used elsewhere (in the NLP entity Dialog form). I suggest you add a new string.

If you could also add the same in fr/translation.json for the french translation (Type goes also for french).